From: Tobias Frost <tobi@debian.org>
Date: Sun, 4 Nov 2018 22:59:09 +0000
Subject: Disable "Returns error when host cannot be resolved"

This patch disables a test that relies on the presence of a
network connection.
---
 test.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test.c b/test.c
index 89f05e1..0f5bfe5 100644
--- a/test.c
+++ b/test.c
@@ -447,11 +447,11 @@ static void test_blocking_connection_errors(void) {
          strcmp(c->errstr,"no address associated with name") == 0));
     redisFree(c);
 
-    /*test("Returns error when the port is not open: ");
+    test("Returns error when the port is not open: ");
     c = redisConnect((char*)"localhost", 1);
     test_cond(c->err == REDIS_ERR_IO &&
         strcmp(c->errstr,"Connection refused") == 0);
-    redisFree(c);*/
+    redisFree(c);
 
     test("Returns error when the unix_sock socket path doesn't accept connections: ");
     c = redisConnectUnix((char*)"/tmp/idontexist.sock");
